main.py

```diff
@@ -22,129 +42,149 @@ def main():
-                                                .output(outpath, vcodec = "libx264", pix_fmt = "yuv420p", acodec = "aac", crf = "23")
+                                                .input(input_file)
+                                                .trim(start = in_point, end = out_point)
+                                                .output(output_path, vcodec = "libx264", pix_fmt = "yuv420p", acodec = "aac", crf = "23")
                                                 .run()
                                         )
                 else:
                         print("ERROR: Path not valid.")
 
-# Function to parse the trim time from the given string.
-def parseTime(timeString):
-        # Splits input time into format [hours, minutes, [seconds, milliseconds]].
-        return [val.split('.') for val in timeString.split(':')]
-
 # Function to get the DAR from a given aspect ratio string.
-def getDAR(aspectRatio):
-        vals = [int(val) for val in aspectRatio.split(':')]
+def display_aspect_ratio(string):
+        vals = [int(val) for val in string.split(':')]
         return vals[0] / vals[1]
 
 # Function to get the aspect ratio and resolution from a given path to a video.
-def getARandRes(path):
-        inputfile = FFProbe(path)
+def aspect_ratio_and_resolution(file):
+        file_metadata = FFProbe(file)
 
-        for stream in inputfile.streams:
+        ratio = None
+        for stream in file_metadata.streams:
                 if stream.is_video():
                         width, height = stream.frame_size()
-                        
-                        ratio = ""
-                        for aspect in aspects:
-                                if aspect[0] == width:
-                                        ratio = aspect[1]
-                                        break
 
-                        if ratio == "":
+                        for resolution in RESOLUTIONS:
+                                if RESOLUTIONS[resolution]['width'] == width:
+                                        ratio = RESOLUTIONS[resolution]['aspect_ratio']
+
+                        if not ratio:
                                 print("Incompatible input dimensions.")
                                 sys.exit()
-                        
+
         return ratio, [width, height]
 
 # Function to get a path to a corresponding .png watermark given an aspect ratio string.
-def getWatermark(ratio):
-        for aspect in aspects:
-                if aspect[1] == ratio:
-                        standard = aspect[2]
-
-        pngname = "{0}.png".format(standard)
-        pngpath = os.path.join(os.getcwd(), "watermarks", pngname)
-        return pngpath
+def watermark_path(ratio):
+        for name, data in RESOLUTIONS.items():
+                if RESOLUTIONS[name]['aspect_ratio'] == ratio:
+                        standard = name # sets standard to "HD", "SD", etc
+
+        file_name = "{0}.png".format(standard)
```

Comment:
If I understood correctly, you have several files named like `HD.png`, `SD.png`, etc?

Reply:
Can do, they're MACE specific but I can generate a generic version for size/structure etc. I'll do that later.

Reply:
Or even add your MACE ones! Up to you, though.